docs/guides/src/main/server/configuration_overview.adoc

```diff
@@ -0,0 +1,173 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Configuring Keycloak"
+summary="An overview of the server configuration">
+
+In this guide, you will learn the core concepts around the server configuration and how to configure the server using the different
+configuration options available. You will also learn how to properly configure the server to achieve an optimal runtime for faster
+startup and low memory footprint.
+
+== Understanding Server Options
+
+All the available configuration options can be found by either running the `help` command for individual commands:
+
+<@kc.all parameters="--help"/>
+
+Or by looking at all the configuration options at <@links.server id="all-config"/>.
```

Comment:
"Or" is a conjuction in a sentence. 
Change to "You can instead look at all configuration options at <@links.server id="all-config"/>."